lms/envs/pact.py

```diff
@@ -11,4 +11,9 @@
 MOCK_USERNAME = 'Mock User'
 
 ######################### Add Authentication Middleware for Pact Verification Calls #########################
-MIDDLEWARE = MIDDLEWARE + ['common.test.pacts.middleware.AuthenticationMiddleware', ]
```

Comment:
I'd like to rename this previous middleware as well, for the reasons discussed here: https://github.com/openedx/edx-platform/pull/33000#discussion_r1293832055